crates/quickjs-wasm-rs/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Callback functions now work with `JSContextRef` again and the `CallbackArg::try_into` conversions are implemented for it instead.
```

Comment:
We also need an entry added to the changed section for the signature of `from_qjs_value` changing since it now takes ownership of the `JSValueRef` instead of a reference.